docs/cudf/source/Working-with-missing-data.ipynb

```diff
@@ -0,0 +1,3032 @@
+{
```

Comment:
<p>Similar as above </p><pre class="ql-syntax" spellcheck="false">&gt;&gt;&gt; pd.Series([np.nan], dtype='float64').prod(skipna=False)
nan
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cudf/pull/7010/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>

Reply:
<p>Same here, added this example too. We seem to be in-line with pandas behavior actually.</p>